python/mxnet/contrib/tensorboard.py

```diff
@@ -71,3 +72,43 @@ def __call__(self, param):
             if self.prefix is not None:
                 name = '%s-%s' % (self.prefix, name)
             self.summary_writer.add_scalar(name, value)
+
+    def node_histogram_visualization(self, prefix=None, node_names=None, bins="auto"):
```

Comment:
How is this supposed to be used? Should it be a new class?

Reply:
Thank you for comment.
This method can be used like below.

```python
lm = mx.contrib.tensorboard.LogMetricsCallback("/log")
model.fit(train,
...
    epoch_end_callback=lm.node_histogram_visualization(),
...
)
```

I understand it would be nice that histogram callback is defined as a new class.
But if you use both of scalar and histogram callback, you should use the same `tensorboard.SummaryWriter` instance.
Another way, make a new class and give SummaryWriter instance as an argument like below.

```python
class HistogramCallback(object):
    def __init__(self, logging_dir, prefix=None, summarywriter=None):
```

Any ideas?